sky/shell/platform/ios/sky_dynamic_service_loader.mm

```diff
@@ -156,18 +182,18 @@ - (SkyDynamicServiceHandler)serviceEntryPoint {
   }
 
   dlerror();
-  void* entry = dlsym(_libraryHandle, kFlutterServiceMainFunctionName);
+  void* entry = dlsym(_libraryHandle, kFlutterServiceInvokeProcName);
 
   if (entry == NULL || dlerror() != NULL) {
     LOG(INFO) << "Could not find service entry point '"
```

Comment:
I think it'd be cleaner to look up all entry points on initially loading and not call in to the DLL until you know that all of the symbols you might want to invoke are exported. you'll still run dll initializers inside the dll before checking the symbols, which you can't avoid (unless you patch the linker itself) but you can simplify error handling a lot by avoid having partially-initialized but invalid libraries loaded